### PR TITLE
Fix build with SSE4.1 support

### PR DESCRIPTION
**Pull request by cho-m**
Seen while building `gst-plugins-rs` in Homebrew https://github.com/Homebrew/homebrew-core/pull/116170

Homebrew's Intel macOS binaries are built for Nehalem, which supports SSE4.1. The build failed on some missing symbols for SSE4.1:
```
= note: Undefined symbols for architecture x86_64:
              "_VP8LDspInitSSE41", referenced from:
                  _VP8LDspInit in liblibwebp_sys-30ac17e44a2197bd.rlib(lossless.o)
            ld: symbol(s) not found for architecture x86_64
            clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

Looks like function is implemented in `src/dsp/lossless_sse41.c`: https://chromium.googlesource.com/webm/libwebp/+/b0a8608/src/dsp/lossless_sse41.c#122

### CODE REVIEW

**Comment by nirbheek** [comment]
This will fix https://github.com/qnighy/libwebp-sys2-rs/issues/12.

**Comment by Randall-Coding** [comment]
Note to that maintainer, libwebp-rs will need to update it's libwebp-sys2 dependency after this PR is merged for it to take effect there also.  That crate is currently using version `0.1.0` https://github.com/qnighy/libwebp-rs/blob/master/Cargo.toml

**Comment by qnighy** [comment]
Sorry for the late response. I will look at it sometime.

**Review by qnighy — APPROVED** [review]
Looks nice. Thank you!

**Comment by qnighy** [comment]
> Note to that maintainer, libwebp-rs will need to update it's libwebp-sys2 dependency after this PR is merged for it to take effect there also. That crate is currently using version `0.1.0` https://github.com/qnighy/libwebp-rs/blob/master/Cargo.toml

No, Cargo interprets `0.1.0` as `^0.1.0`.

**Pull request merged**
